deployments/ansible/README.md

```diff
@@ -0,0 +1,57 @@
+# SignalFx Agent Ansible Playbook
```

Comment:
I think this is a "role".  Playbooks are what specify your infrastructure as a whole right?  Confirm this terminology.

Reply:
Yes, it is the 'signalfx-agent' role defined in the playbook which will play in the defined infrastructure. Ideally customer going to copy our role to their playbook and play in their infra to install signalfx agent. Updated the doc accordingly.